src/core/theme/index.ts

```diff
@@ -47,11 +48,13 @@ export const defaultTokens = {
  * @param tokens SuomifiTokens, defaults to suomifi-tokens
  */
 export const suomifiTheme = ({
-  colors,
-  spacing,
-  typography,
-}: SuomifiTokens = defaultTokens) => ({
+  colors = defaultTokens.colors,
+  spacing = defaultTokens.spacing,
+  typography = defaultTokens.typography,
+  ...internalTokensProp
+}: Partial<SuomifiTokens & DefaultInternalTokens> = defaultTokens) => ({
   ...internalTokens,
+  ...internalTokensProp,
```

Comment:
What is this and where is this used / where does this come from?

Reply:
This is the internal tokens that are customized. You can highlight it with IDE and typescript will tell what it contains. I'll update JSDOC for the function

Reply:
Great. That's what I figured as well, but updating the JSDOC will definitely make it easier to see.

Reply:
A bit difficult to follow what instances / types get returned, maybe rename internalTokensProp to be more descriptive when considering the use. In addition, documenting the return values and types might help.